xbmc/utils/StreamDetails.cpp

```diff
@@ -484,6 +490,24 @@ std::string CStreamDetails::GetSubtitleLanguage(int idx) const
     return "";
 }
 
+std::string CStreamDetails::GetSubtitleCodec(int idx) const
+{
+  const CStreamDetailSubtitle* item =
+      dynamic_cast<const CStreamDetailSubtitle*>(GetNthStream(CStreamDetail::SUBTITLE, idx));
+  if (item)
+    return item->m_type;
```

Comment:
```suggestion
    return item->m_codec;
```
And the reverse in the function below?